src/components/ListJobs.vue

```diff
@@ -0,0 +1,63 @@
+<template>
```

Comment:
Can you create a folder list-jobs and move the component there? It would be nice consistency with the rest components

Reply:
done